features/networking/sriov.feature

```diff
@@ -14,3 +14,58 @@ Feature: Sriov related scenarios
       | app=sriov-network-config-daemon |
     And status becomes :running of exactly 1 pods labeled:
       | name=sriov-network-operator     |
+
+
+  # @author zzhao@redhat.com
+  # @case_id OCP-24702
+  @destructive
+  @admin
+  Scenario: netdevice VF for XXV710 can be worked well when sriovnetworknodepolicies is created for rhcos node
+    Given the sriov operator is running well
+    #When I run the :create admin command with:
+    #  | f | <%= BushSlicer::HOME %>/testdata/networking/sriov/sriovnetworkpolicy/intel-netdevice.yaml |
+    #Then the step should succeed
+    Given I create resource with following:
+       | cr_yaml       |  <%= BushSlicer::HOME %>/testdata/networking/sriov/sriovnetworkpolicy/intel-netdevice.yaml |
```

Comment:
@zhaozhanqi pls use `Given I obtain...` to get the testdata going forward.

Reply:
thanks @pruan-rht updated.